packages/contracts/contracts/ArtonomousArtPieceToken.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.4.23;
+
+import "openzeppelin-solidity/contracts/token/ERC721/ERC721Token.sol";
+
+contract ArtonomousArtPieceToken is ERC721Token {
```

Comment:
Just a note, should be owned by the `Artonomous` contract but we can update that later.